packages/ra-core/src/createAdminStore.ts

```diff
@@ -61,11 +60,7 @@ export default ({
         resettableAppReducer,
         initialState,
         compose(
-            applyMiddleware(
-                sagaMiddleware,
-                formMiddleware,
-                routerMiddleware(history)
-            ),
+            applyMiddleware(sagaMiddleware, routerMiddleware(history)),
```

Comment:
this is a breaking change for custom apps, it should be documented in the custom apps and the upgrade guide

Reply:
Done